client/src/main/scala/org/apache/celeborn/client/LifecycleManager.scala

```diff
@@ -862,8 +862,16 @@ class LifecycleManager(val appUniqueId: String, val conf: CelebornConf) extends
       return
     }
 
-    def isAllMaptaskEnd(shuffleId: Int): Boolean = {
-      !commitManager.getMapperAttempts(shuffleId).exists(_ < 0)
+    def isAllMapTasksEnd(shuffleId: Int): Boolean = {
```

Comment:
Seems we can eliminate these duplicate code.

Reply:
done